inspirehep/modules/records/mappings/records/authors.json

```diff
@@ -2,7 +2,7 @@
     "mappings": {
         "authors": {
             "_all": {
-                "enabled": false
+                "enabled": true
```

Comment:
This will probably be reverted, because we expect people to want high relevance when searching for people.